### PR TITLE
add dim_labels

### PR DESCRIPTION
**Pull request by tarikceric**
Add `dim_labels`

- FR
```
23:44:55  1 of 2 OK created sql incremental model silver.labels .......................... [SUCCESS 1 in 4.94s]
23:44:57  2 of 2 OK created sql view model core.dim_labels ............................... [SUCCESS 1 in 2.39s]
```

- Test
`23:46:28  Finished running 9 data tests, 5 project hooks in 0 hours 0 minutes and 25.61 seconds (25.61s).`

### CODE REVIEW

File: models/gold/core/core__dim_labels.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          - dbt_expectations.expect_column_to_exist
+      - name: LABEL_TYPE
+        description: "{{ doc('label_type') }}"
+        tests:
+          - dbt_expectations.expect_column_to_exist
+      - name: LABEL_SUBTYPE
+        description: "{{ doc('label_subtype') }}"
+        tests:
+          - dbt_expectations.expect_column_to_exist
+      - name: LABEL
```
Comment (desmond-hui) [inline models/gold/core/core__dim_labels.yml:30]: missing `address_name` in the schema

File: models/silver/labels/silver__labels.sql [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{{ config(
+    materialized = 'incremental',
+    unique_key = ['address', 'blockchain'],
+    incremental_strategy = 'merge',
+    merge_exclude_columns = ["inserted_timestamp"],
+    cluster_by = 'modified_timestamp::DATE',
+    post_hook = "ALTER TABLE {{ this }} ADD SEARCH OPTIMIZATION ON EQUALITY(address); DELETE FROM {{ this }} WHERE _is_deleted = TRUE;",
```
Comment (desmond-hui) [inline models/silver/labels/silver__labels.sql:7]: use `enable_search_optimization` macro instead

File: models/silver/labels/silver__labels.sql [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    address,
+    creator,
+    label_type,
+    label_subtype,
+    address_name,
+    project_name,
+    _is_deleted,
+    labels_combined_id,
+    SYSDATE() AS inserted_timestamp,
+    SYSDATE() AS modified_timestamp,
```
Comment (desmond-hui) [inline models/silver/labels/silver__labels.sql:22]: nit: lowercase function names

File: models/silver/labels/silver__labels.sql [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        'crosschain_silver',
+        'labels_combined'
+    ) }}
+WHERE
+    blockchain = 'eclipse'
+
+{% if is_incremental() %}
+AND modified_timestamp >= (
+    SELECT
+        MAX(
```
Comment (desmond-hui) [inline models/silver/labels/silver__labels.sql:35]: nit: lowercase function names

File: models/silver/labels/silver__labels.yml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+models:
+  - name: silver__labels
+
+    columns:
+      - name: BLOCKCHAIN
+        tests:
+          - not_null  
+      - name: ADDRESS
+        tests:
+          - not_null
```
Comment (desmond-hui) [inline models/silver/labels/silver__labels.yml:11]: Where are the rest of the column definitions?

**Review by desmond-hui — CHANGES_REQUESTED** [review]

File: models/gold/core/core__dim_labels.yml (hunk 1 above)
Comment (tarikceric) [reply models/gold/core/core__dim_labels.yml:30]: updated

File: models/silver/labels/silver__labels.sql (hunk 2 above)
Comment (tarikceric) [reply models/silver/labels/silver__labels.sql]: updated

File: models/silver/labels/silver__labels.sql (hunk 3 above)
Comment (tarikceric) [reply models/silver/labels/silver__labels.sql]: updated

File: models/silver/labels/silver__labels.sql (hunk 4 above)
Comment (tarikceric) [reply models/silver/labels/silver__labels.sql]: updated

File: models/silver/labels/silver__labels.yml (hunk 5 above)
Comment (tarikceric) [reply models/silver/labels/silver__labels.yml]: updated with rest of cols + tests

**Comment by tarikceric** [comment]
Actually, should we be making the `dim_labels` view a table since thats we want for all gold?

File: models/gold/core/core__dim_labels.sql [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    project_name AS label,
+    address_name,
+    labels_combined_id AS dim_labels_id,
+    inserted_timestamp,
+    modified_timestamp
+FROM
+    {{ ref('silver__labels') }}
+
+{% if is_incremental() %}
+AND modified_timestamp >= (
```
Comment (eric-laurello) [inline models/gold/core/core__dim_labels.sql:25]: bad inc logic

**Review by eric-laurello — APPROVED** [review]

**Review by desmond-hui — APPROVED** [review]

**Pull request merged**